python/ray/autoscaler/aws/cloudwatch/example-cloudwatch-agent-config.json

```diff
@@ -8,6 +8,74 @@
       "port":31000
    },
    "logs":{
+      "metrics_collected": {
+        "prometheus": {
+          "log_group_name": "{cluster_name}-Prometheus",
+          "prometheus_config_path": "/opt/prometheus-2.25.0.linux-amd64/prometheus.yml",
+          "emf_processor": {
+            "metric_declaration_dedup": true,
+            "metric_namespace": "{cluster_name}-Prometheus",
+            "metric_unit":{
+              "python_gc_collections_total": "Count",
```

Comment:
We should create a follow-up issue to investigate methods of removing the hard-coding of this {METRIC_NAME}: {METRIC_UNIT} mapping. Ideally, it seems like all of these metric names could be changed to follow the base units metric naming convention at: https://prometheus.io/docs/practices/naming/ (with the "_count" suffix ideally used for any metric that is just counting occurrences of something). It looks like metric units should also available for Prometheus counter metric types: https://github.com/ray-project/ray/blob/master/python/ray/_private/prometheus_exporter.py#L157.

Reply:
Agreed. one way to remove hard-coding (at least for raylet and GCS related metrics) is to retrive the metric name & type via https://github.com/ray-project/ray/blob/master/src/ray/stats/metric_defs.h

Reply:
Thanks Patrick and Anna for suggestions. Will look into automatic metric unit conversion as future improvement.